hw/ip/i2c/README.md

```diff
@@ -26,12 +26,12 @@ See that document for integration overview within the broader top level system.
     - No support for host-host clock synchronization
     - No support for host bus arbitration.
 - Byte-formatted register interface with four separate queues: two queues in the host mode, one for holding read data, the other for holding bytes to be transmitted (addresses or write data) and two queues in the target mode, for holding read and write data
-- Direct SCL and SDA control in "Override mode" (for debugging)
-- SCL and SDA ports mapped to I/O via the pinmux.
-- Interrupts in the host mode for FMT and RX FIFO overflow, target NACK, SCL/SDA signal interference, timeout, unstable SDA signal levels, and transaction complete
-- Interrupts in the target mode for TX FIFO empty during a read, TX FIFO nonempty at the end of a read, TX overflow and ACQ FIFO full, host sending STOP after ACK, and host ceasing to send SCL pulses during an ongoing transaction
-- Loopback support with external host when in target operation.
-- SW may reset I2C block using the Reset Manager.
+- Direct SCL (serial clock line) and SDA (serial data line) control in "Override mode" (for debugging)
+- SCL and SDA ports mapped to I/O via the pinmux
+- Interrupts in the host mode for FMT and RX (receive) FIFO overflow, target NACK, SCL/SDA signal interference, timeout, unstable SDA signal levels, and transaction complete
```

Comment:
I think `(receive)` and `(transmit)` are redundant here.

Line 28 refers to a `register interface with four separate queues`, maybe we should update the language there to TX and RX FIFOs instead of queues?

Reply:
I left the queues terminology because I couldn't figure out how to reword it without making it sound more complicated. I did add RX and TX to line 28 and removed it from line 31.